subsys/net/l2/wifi/Kconfig

```diff
@@ -125,3 +125,7 @@ config WIFI_ENT_IDENTITY_MAX_USERS
 	default 8
 	help
 	  This option defines the maximum number of identity users allowed connection.
+
+config SHELL_ARGC_MAX
+	int
```

Comment:
do we need this type definition, isn't it just an override?

Reply:
Indeed I don't this works in the Kconfig syntax.
https://docs.kernel.org/kbuild/kconfig-language.html
https://github.com/zephyrproject-rtos/zephyr/blob/c4ebf713b10ce63f44ee3497159c54377ecc839b/subsys/shell/Kconfig#L46-L51
Maybe check on how they do it for other symbols

Reply:
> do we need this type definition, isn't it just an override?

We don't really need to define the type here, so I have updated the PR.